package.json

```diff
@@ -40,27 +40,26 @@
     "biscotto": "wizzy25/biscotto#v2.4.1",
     "bluebird": "2.9.x",
     "coffeelint": "^1.14.1",
-    "commitizen": "2.5.0",
-    "coveralls": "2.11.x",
+    "commitizen": "^3.0.4",
+    "coveralls": "^3.0.2",
     "cz-conventional-changelog": "1.1.5",
     "express": "4.16.x",
-    "gh-pages": "0.2.x",
+    "gh-pages": "^2.0.1",
     "ghooks": "1.0.3",
-    "grunt": "0.4.5",
-    "grunt-bump": "0.0.13",
+    "grunt": "^1.0.3",
+    "grunt-bump": "^0.8.0",
     "grunt-cli": "^1.2.0",
     "grunt-coffeelint": "0.0.13",
     "grunt-contrib-clean": "0.6.0",
-    "grunt-contrib-coffee": "0.13.x",
+    "grunt-contrib-coffee": "github:daern91/grunt-contrib-coffee",
```

Comment:
Changing this one to forked version with updated dependencies. Reason for this is that `v1.0.0` works for us but has one warning. In addition, `v2.x.x` fixed this but also upgraded to use `CoffeeScript v2` which introduced a lot of [breaking changes](https://coffeescript.org/v2/#breaking-changes)

This did not work out for us and made all [tests fail](https://travis-ci.org/sphereio/sphere-node-sdk/jobs/457526587)